src/libraries/System.Diagnostics.Process/tests/ProcessStartInfoTests.cs

```diff
@@ -467,63 +466,15 @@ public void TestWorkingDirectoryPropertyInChildProcess()
         [OuterLoop("Requires admin privileges")]
         public void TestUserCredentialsPropertiesOnWindows()
         {
-            const string username = "testForDotnetRuntime";
-            using WindowsTestAccount testAccount = new WindowsTestAccount(username);
-
-            bool hasStarted = false;
-            SafeProcessHandle handle = null;
-            Process p = null;
-            string workingDirectory = null;
-
-            try
-            {
-                p = CreateProcessLong();
-
-                workingDirectory = string.IsNullOrEmpty(p.StartInfo.WorkingDirectory)
-                    ? Directory.GetCurrentDirectory()
-                    : p.StartInfo.WorkingDirectory;
-
-                if (PlatformDetection.IsNotWindowsServerCore) // for this particular Windows version it fails with Attempted to perform an unauthorized operation (#46619)
-                {
-                    // ensure the new user can access the .exe (otherwise you get Access is denied exception)
-                    SetAccessControl(username, p.StartInfo.FileName, workingDirectory, add: true);
-                }
-
-                p.StartInfo.LoadUserProfile = true;
-                p.StartInfo.UserName = username;
-                p.StartInfo.PasswordInClearText = testAccount.Password;
-
-                try
-                {
-                    hasStarted = p.Start();
-                }
-                catch (Win32Exception ex) when (ex.NativeErrorCode == ERROR_SHARING_VIOLATION)
-                {
-                    throw new SkipTestException($"{p.StartInfo.FileName} has been locked by some other process");
-                }
+            using Process p = CreateProcessLong();
```

Comment:
nit: since we started the refactoring, we can use some more self-describing name

```suggestion
            using Process longRunning = CreateProcessLong();
```